sdk/purview/azure-purview-datamap/samples/get_types.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+from azure.identity import ClientSecretCredential
+from azure.purview.datamap import DataMapClient
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+try:
+    endpoint = os.environ["PURVIEW_ENDPOINT"]
+    tenantId = os.environ["PURVIEW_TENANT_ID"]
+    clientId = os.environ["PURVIEW_CLIENT_ID"]
+    clientSecret = os.environ["PURVIEW_CLIENT_SECRET"]
+    authority = os.environ["AUTHORITY_HOST"] #e.g"https://login.microsoftonline.com/"
```

Comment:
Do you need to set this?

Reply:
Customers can easily get this value?

Reply:
Are customers mainly using the df environment or the prod env?

Reply:
Wont block this PR on this question since you've tested this on your end, but if this is only necessary for specific scenarios I would recommend having a specific sample for those environments OR add a comment to explain when this environment variable should be used and to make this more easily consumable for customers you could add the default value for this environment variable for folks.

Reply:
Thanks for the suggestion. We will improve this as the follow-up items